doc/source/ray-observability/getting-started.rst

```diff
@@ -311,7 +311,7 @@ Additionally, users can see a snapshot of hardware utilization from the :ref:`Cl
 View the resource utilization
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Ray requires users to specify the number of :ref:`resources <logical-resources>` their Tasks and Actors to use through arguments such as ``num_cpus``, ``num_gpus``, ``memory``, and ``resource``.
+Ray requires users to specify the number of :ref:`resources <logical-resources>` their Tasks and Actors need to use through arguments such as ``num_cpus``, ``num_gpus``, ``memory``, and ``resource``.
```

Comment:
```suggestion
Ray requires users to specify the number of :ref:`resources <logical-resources>` their Tasks and Actors use through arguments such as ``num_cpus``, ``num_gpus``, ``memory``, and ``resource``.
```

Reply:
I think this is the desired direction to edit?

Reply:
LGTM